pkg/hcl_interpreter/errors.go

```diff
@@ -0,0 +1,29 @@
+// © 2022-2023 Snyk Limited All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hcl_interpreter
+
+import (
+	"errors"
+)
+
+var ErrLoadRemoteSubmodules = errors.New("Could not load some remote submodules")
+
+var ErrEvaluation = errors.New("Evaluation error")
+
+var ErrUnhandledValueType = errors.New("Unhandled value type")
+
+var ErrBadDependencyKey = errors.New("Bad dependency key")
```

Comment:
This error and UnhandledValueType should probably not be exported, they are purely internal and if we see one it’s a policy engine bug.

Reply:
That makes sense. I think both of those end up in the list of non-fatal errors atm, so we'll need to change that.